source/connect/connection-options.txt

```diff
@@ -0,0 +1,286 @@
+.. _cpp-connection-options:
+
+==========================
+Specify Connection Options
+==========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: connection string, URI, server, Atlas, settings, configure
+
+Overview
+--------
+
+This page describes the MongoDB connection and authentication options
+available in the {+driver-short+}. You can configure your connection by using
+options in the connection URI.
+
+.. note::
+
+   You can set most connection options by using either the connection URI or the
+   ``client_options`` parameter. However, some options support only one of these
```

Comment:
S: I think it's helpful to specify that this parameter should be an instance of mongocxx::options::client
```suggestion
  a ``mongocxx::options::client`` instance as a parameter. However, some options support only one of these
```